django/db/models/options.py

```diff
@@ -354,6 +336,21 @@ def swapped(self):
                     return swapped_for
         return None
 
+    @property
```

Comment:
Thought about this, but then it would go stale should someone patch a class with a manager (I think people do that with managers). It's mostly used at init to figure out the default manager and when creating migration states, so it's probably not too bad.